utils/__tests__/messages.test.ts

```diff
@@ -137,15 +162,91 @@ describe("Messages", () => {
       expect(E.isRight(enrichedMessage)).toBe(true);
       if (E.isRight(enrichedMessage)) {
         expect(EnrichedMessage.is(enrichedMessage.right)).toBe(true);
+        expect(enrichedMessage.right.category).toEqual({
+          tag: TagEnumBase.GENERIC
+        });
+      }
+    });
+    expect(functionsContextMock.log.error).not.toHaveBeenCalled();
+  });
+
+  it("should return right with right message GREEN_PASS category when message content is retrieved", async () => {
```

Comment:
Not sure about your own conventions, but on the io-app we are pursuing a more structured (and somehow rigid) style for testing. See [this example](https://github.com/pagopa/io-app/blob/a8c55a2ead0f9d3d7bf019a558bc82ad15b8164a/ts/store/reducers/entities/messages/__tests__/allPaginated.test.ts#L26). 
It is loosely inspired by [given-when-then](https://martinfowler.com/bliki/GivenWhenThen.html).

Reply:
This could be a nice convention, I'll check it out for future tests.